gulpfile.js

```diff
@@ -69,3 +73,33 @@ function isWin() {
 function mvnw() {
     return isWin() ? 'mvnw.cmd' : './mvnw';
 }
+
+function updateVersion() {
+    // Update the version
+    const packageJsonData = require('./package.json');
+    const javaExtensions = packageJsonData.contributes.javaExtensions;
+    if (Array.isArray(javaExtensions)) {
+        packageJsonData.contributes.javaExtensions  = javaExtensions.map((extensionString) => {
+            
+            const ind = extensionString.indexOf('_');
+            const fileName = findNewPDEPlugin(extensionString.substring(extensionString.lastIndexOf('/') + 1, ind));
+            
+            if (ind >= 0) {
+                return extensionString.substring(0, extensionString.lastIndexOf('/') + 1) + fileName;
+            }
+            return extensionString;
+        });
+
+        fs.writeFileSync('./package.json', JSON.stringify(packageJsonData, null, 4));
+    }
+}
+
+function findNewPDEPlugin(fileName) {
```

Comment:
should better add short comments for it, about the implication that PDE plugins are named as `<name>_<version>.jar`